.teamcity/settings/root-project.kt

```diff
@@ -7,6 +7,7 @@ fun Project.configureGradleProfilerProject() {
     buildType(MacOSJava18)
     buildType(WindowsJava18)
     buildType(LinuxJava18)
+    buildType(LinuxJava11)
```

Comment:
I think this would be a good opportunity to rename `Java18` to `Java8`.

Reply:
I can do this post-merge, so I still run the tests on my branch. Renaming the object causes a change in the ID in Teamcity, so I can't do this change on a branch.

Reply:
Done: https://github.com/gradle/gradle-profiler/commit/53c13e43cab0935424adb65fd4ce0cf2bc199792